### PR TITLE
Use bmake compatible syntax where possible for all targets but install

### PR DESCRIPTION
**Pull request by bapt**
Note that the install part is not working as expected yet with bmake, but at least it is now possible to build everything with bmake

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,30 +7,22 @@ datarootdir=$(prefix)/share
...
-else
-	LIBSRC = libintl/libintl.c
-	HEADERS = libintl.h
-endif
-PROGSRC = $(sort $(wildcard src/*.c))
+-include libintl-$(LIBINTL).mak
+LIBSRC ?= libintl/libintl.c
+HEADERS ?= libintl.h
+
+PROGSRC != ls src/*.c
```
Comment (rofl0r) [inline Makefile:14]: what does that syntax mean? is it compatible with gnu make ?

File: Makefile [hunk 2]
```diff
@@ -7,30 +7,22 @@ datarootdir=$(prefix)/share
...
 
 ALL_INCLUDES = $(HEADERS)
-ifneq ($(LIBINTL), NONE)
-ALL_LIBS=libintl.a
-endif
+ALL_LIBS ?= libintl.a
 ALL_TOOLS=msgfmt msgmerge xgettext autopoint
-ALL_M4S=$(sort $(wildcard m4/*.m4))
-ALL_DATA=$(sort $(wildcard data/*))
+ALL_M4S!=	ls m4/*.m4
```
Comment (rofl0r) [inline Makefile:24]: ok urgh this is running shell command and ls output isn't portable. i would rather find another solution to this, even when it means hardcoding all filenames.

File: README.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ make LIBINTL=FLAVOR
 make LIBINTL=FLAVOR DESTDIR=pkgdir prefix=/ install
 ```
 
-where FLAVOR can be one of NONE, MUSL, NOOP (as detailed above).
+where FLAVOR can be one of "none", "musl", NOOP (as detailed above).
```
Comment (rofl0r) [inline README.md:45]: what's the reason to change this to lowercase ?

File: Makefile (hunk 2 above)
Comment (xhebox) [reply Makefile:24]: Compatible. But I like the GNU one, which seems more portable to me(relatively speaking?). So is gnu make more common or `ls` more common...?

**Pull request closed without merging**